projects/common-platform-and-delius/build.gradle.kts

```diff
@@ -16,8 +16,10 @@ dependencies {
     implementation("org.jetbrains.kotlin:kotlin-reflect")
     implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
     implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
+    implementation("com.fasterxml.jackson.dataformat:jackson-dataformat-csv")
     implementation("org.opensearch.client:spring-data-opensearch-starter:1.6.1")
     implementation("org.opensearch.client:opensearch-java:2.20.0")
+    implementation("io.awspring.cloud:spring-cloud-aws-starter-s3:3.3.0")
```

Comment:
Could we move these three dependency versions in to the versionCatalogs section in settings.gradle.kts?